web/router.go

```diff
@@ -54,7 +57,7 @@ func NewRouter(settings RouterSetting) (model.Router, error) {
 		r.AdminRouter, err = admin.NewRouter(
 			settings.Server,
 			settings.Logger,
-			settings.AdminRouterSettings...,
+			append(settings.AdminRouterSettings, admin.WebRouterPrefixOption(r.WebRouterPath))...,
```

Comment:
Add web router prefix for Admin

Reply:
I am about to refactor the web router and login router completely, It will be definitely conflicting with this implementation.

Could you please provide me with more details on what have you done here, so I can apply it to the new implementation as well. Otherwise we will hit the same issue again.

Reply:
I have added web router prefix to admin router to allow admin generate link to reset password without user request.